apis/nucleus/src/index.js

```diff
@@ -416,7 +416,7 @@ function nuked(configuration = {}) {
            * @param {string=} options.title Custom title, defaults to fieldname
            * @param {Direction=} [options.direction=ltr] Direction setting ltr|rtl.
            * @param {ListLayout=} [options.listLayout=vertical] Layout direction vertical|horizontal
-           * @param {FrequencyMode=} [options.frequencyMode=none] Show frequency none|value|percent|relative
+           * @param {FrequencyMode=} [options.frequencyMode=N] Show frequency none|value|percent|relative
```

Comment:
This is a little bit confusing, because the available values for `FrequencyMode` are:
none, value, percent and relate according to the spec:
```
"FrequencyMode": {
      "kind": "alias",
      "items": {
        "kind": "union",
        "items": [
          {
            "kind": "literal",
            "value": "'none'"
          },
          {
            "kind": "literal",
            "value": "'value'"
          },
          {
            "kind": "literal",
            "value": "'percent'"
          },
          {
            "kind": "literal",
            "value": "'relative'"
          }
        ]
      }
    },
```

Reply:
So I think you can just keep this as `none`... Sure, the corresponding property might be `N`, but the option is still `none` the way I see it.